style.css

```diff
@@ -0,0 +1,119 @@
+
+body{
+    background-color:#9B90A0;
+    font-family:"Veranda", sans-serif;
+
+}
+ul {
+    list-style-type: none;
+    margin:0;
+    padding:0;
+    overflow: hidden;
+    background-color:#DEBEC2;
+
+}
+h1{
+    background-image: url("IMG_7088.jpg");
```

Comment:
This isn't a problem yet, but it can get a bit messy having all your images and all your php files in the same directory. You might try making an "images" folder, and then this line would look like:
```
 background-image: url("/images/IMG_7088.jpg");
```

Note that the first "/" says "go to the top level of the project, and then look in the images folder". If you left that out, it would look in the images folder inside the folder you're already in. This way, if one day you have a file with the path `/blog/admin/edit_post.php` it'll still be able to look in the main images folder. Let me know if that doesn't make sense.